ifbcat_api/model/tool/tool.py

```diff
@@ -44,6 +44,9 @@ class Meta:
     )
     tool_credit = models.ManyToManyField(ToolCredit, blank=True)
     tool_license = models.CharField(max_length=1000, blank=True, null=True)
+    documentation = models.URLField(
+        max_length=255, null=True, blank=True, help_text="Link toward general documentation of the tool"
```

Comment:
I'm wondering if 255 is enough. We should observe in the tools we already have the max length for a doc, and maybe double ortripple it

Reply:
I checked the max size of items in db and the max I got was 53. I setted the `max_length` to 512 just in case...